README.md

```diff
@@ -166,8 +230,8 @@ Serve the `OUTPUT_DIR` directory using Python's built-in HTTP server, plus a cou
   - Custom HTTP error pages are emulated, if they are found saved as `{status code}.html` in the output directory
   - The server will listen to both IPv4 *and* IPv6 loopback addresses if possible
```

Comment:
RSS and Atom files will be served using the appropriate MIME types in `Content-Type` response header